spring-credhub-cloud-connector/src/main/java/org/springframework/credhub/cloud/CredHubInterpolationServiceDataPostProcessor.java

```diff
@@ -42,7 +44,21 @@
 	 */
 	public CredHubInterpolationServiceDataPostProcessor() {
 		try {
-			credHubOperations = new CredHubConfiguration().credHubTemplate();
+			CredHubTemplateFactory credHubTemplateFactroy = new CredHubTemplateFactory();
+			CredHubProperties credHubProperties = new CredHubProperties();
+
+			credHubProperties.setUrl(System.getProperty("credhub.url"));
+
+			if (credHubProperties.getUrl() != null
+					&& !credHubProperties.getUrl().isEmpty()) {
+				credHubOperations = credHubTemplateFactroy.credHubTemplate(
+						credHubProperties,
+						credHubTemplateFactroy.clientHttpRequestFactoryWrapper());
+			}
+			else {
+				logger.log(Level.WARNING,
+						"System property spring.credhub is undefined. CredHubOperations cannot be initialized, disabling processing of service data");
```

Comment:
This message doesn't match the property retrieved above (`credhub.url`). Should both of these be `spring.credhub.url`?

Reply:
Typo error indeed. Should be `spring.credhub.url`.